libutils/misc_lib.h

```diff
@@ -73,6 +73,8 @@
 
 #define ISPOW2(n)     (  (n)>0  &&  ((((n) & ((n)-1)) == 0))  )
 
+#define ABS(n)        (  (n<0)  ?  (-n)  :  (n)  )
```

Comment:
Note that `ABS(somefunc())` will cause `somefunc()` to be called 3 times - which can be significant if it's slow. Not much to do about it in C, since we don't know the type, but useful to know / pay attention to. (in C++ we could create a temp variable with `auto` type, I think)

Reply:
Intresting @olehermanse . Never thought about that, but makes sense. Thanks for the heads up 👍